pom.xml

```
@@ -31,7 +31,7 @@
     <changelist>999999-SNAPSHOT</changelist>
     <spotbugs.skip>true</spotbugs.skip>
 
-    <jenkins.version>2.411</jenkins.version>
+    <jenkins.version>2.412-rc33904.b_b_78b_7e315e8</jenkins.version>
```

```suggestion
    <jenkins.version>2.411</jenkins.version>
```
To be removed before merging